test/syntax.vader

```diff
@@ -1303,6 +1303,43 @@ Execute (HTML tag in text):
   AssertEqual SyntaxOf('<span>'), 'htmlTag'
   AssertEqual SyntaxOf('</span>'), 'htmlEndTag'
 
+Given markdown;
+<!--TODO FIXME XXX TBD hello -->
+
+Execute (todo inside HTML comment):
+  AssertEqual SyntaxOf('TODO'), 'mkdTodo'
+  AssertEqual SyntaxOf('FIXME'), 'mkdTodo'
+  AssertEqual SyntaxOf('XXX'), 'mkdTodo'
+  AssertEqual SyntaxOf('TBD'), 'mkdTodo'
+  AssertNotEqual SyntaxOf('hello'), 'mkdTodo'
+
+Given markdown;
+<!--ATODO BFIXME CXXX DTBD -->
+
+Execute (fake todo inside HTML comment):
+  AssertNotEqual SyntaxOf('TODO'), 'mkdTodo'
+  AssertNotEqual SyntaxOf('FIXME'), 'mkdTodo'
+  AssertNotEqual SyntaxOf('XXX'), 'mkdTodo'
+  AssertNotEqual SyntaxOf('TBD'), 'mkdTodo'
```

Comment:
Don't these need to *not* be todos? I believe the word boundary not matching the keyword should disqualify them, no?

Reply:
This part is correct. It checks that the TODO in ATODO and so on aren't todo comments.